dbt-cta/dbt_project.yml

```diff
@@ -40,7 +40,8 @@ models:
       +tags: 
         - partner
       +materialized: materialized_view
-      +auto_refresh: true
+      +auto_refresh: false
+      +full_refresh: true
```

Comment:
This will drop and rebuild matviews right? is that what we want to be doing?

Reply:
yep - we want the refresh to happen as part of the dag run so that the "last modified" metadata in BQ reflects the timing of the sync. And with auto-refresh enabled, refreshing at sync-time means MVs are getting refreshed twice, which is 2x the compute cost we need. It seems like dropping-and-rebuilding matviews doesn't break any views that are built on top of them, so it shouldn't have any ill effects

Reply:
per Slack, just updated full_refresh to be explicitly false!